core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -100,6 +117,11 @@ public int schemaId() {
     public void applyTo(TableMetadata.Builder metadataBuilder) {
       metadataBuilder.setCurrentSchema(schemaId);
     }
+
+    @Override
+    public void applyTo(ViewMetadata.Builder viewMetadataBuilder) {
+      viewMetadataBuilder.setCurrentSchemaId(schemaId);
```

Comment:
Are we going to track `current-schema-id` in views? If not, then we don't need this operation.

Reply:
we won't be tracking this anymore and I opened a follow-up to properly clean this up and to not have additional changes here. See also https://github.com/apache/iceberg/pull/8147#discussion_r1282023957